internal/capability/operator_install.go

```diff
@@ -9,37 +9,37 @@ import (
 
 var logger = log.Sugar
 
-func (ca *CapAudit) OperatorInstall() error {
-	logger.Debugw("installing package", "package", ca.Subscription.Package, "channel", ca.Subscription.Channel, "installmode", ca.Subscription.InstallModeType)
+func (ca *capAudit) OperatorInstall() error {
+	logger.Debugw("installing package", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
 
 	// create operator's own namespace
-	operator.CreateNamespace(context.Background(), ca.Namespace)
+	operator.CreateNamespace(context.Background(), ca.namespace)
 
 	// create remaining target namespaces watched by the operator
-	for _, ns := range ca.OperatorGroupData.TargetNamespaces {
-		if ns != ca.Namespace {
+	for _, ns := range ca.operatorGroupData.TargetNamespaces {
+		if ns != ca.namespace {
 			operator.CreateNamespace(context.Background(), ns)
 		}
 	}
 
 	// create operator group for operator package/channel
-	ca.Client.CreateOperatorGroup(context.Background(), ca.OperatorGroupData, ca.Namespace)
+	ca.client.CreateOperatorGroup(context.Background(), ca.operatorGroupData, ca.namespace)
 
 	// create subscription for operator package/channel
-	_, err := ca.Client.CreateSubscription(context.Background(), ca.Subscription, ca.Namespace)
+	_, err := ca.client.CreateSubscription(context.Background(), ca.subscription, ca.namespace)
```

Comment:
If you're ignoring the first return value, this can be combined with the next line.

Reply:
There we go d9bc1ba